### DISCUSSION TITLE
如何设置UMI4的分包规则

### DISCUSSION THREAD

**Discussion by kGe-z** [umijs/umi#12512] [Q&A]
![image](https://github.com/umijs/umi/assets/47897652/0f3599cb-9ab4-45cb-b961-c9ecf70bae3b)

现在umi4 的 codeSplitting 默认开启 granularChunks 配置，但打包出来的文件过多，如何配置限制分包体积

**Comment by fz6m** [comment]
[代码拆分指南](https://umijs.org/blog/code-splitting)
